internal/cmdinit/cmdinit.go

```diff
@@ -147,21 +151,40 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 	return nil
 }
 
-var manTemplate = `{{.Name}}
-==================================================
+// manTemplate is the content for the automatically generated README.md file.
+// It uses ' instead of ` since golang doesn't allow using ` in a raw string
+// literal. We do a replace on the content before printing.
+var manTemplate = `# {{.Name}}
 
-# NAME
+## Description
+{{.Description}}
```

Comment:
Should we add some meaningful default description as opposed to "sample description"?

Reply:
I'm not sure how useful we can make this. For any serious project users will need to provide the description themselves.